File: week2/c/ayush5.c
#include <stdio.h>
int main() {
    int rows=3, cols=3;
    int matrix[3][3]={{0,0,3},{0,0,0},{7,0,0}};
    int zeros=0;

    for(int i=0;i<rows;i++)
        for(int j=0;j<cols;j++)
            if(matrix[i][j]==0) zeros++;

    if(zeros > (rows*cols)/2)
        printf("Sparse Matrix");
    else
        printf("Not Sparse Matrix");
    return 0;
}

File: week2/py/ayush5.py
matrix = [[0,0,3],[0,0,0],[7,0,0]]
rows, cols = len(matrix), len(matrix[0])
zeros = sum(row.count(0) for row in matrix)

if zeros > (rows*cols)/2:
    print("Sparse Matrix")
else:
    print("Not Sparse Matrix")
